lib/auth/v4/headerAuthCheck.js

```diff
@@ -14,9 +14,11 @@ const headerAuthCheck = {};
  * V4 header auth check
  * @param {object} request - HTTP request object
  * @param {object} log - logging object
+ * @param {object} data - Data of request
```

Comment:
Please be more specific here so we know how this differs from the request parameter above.